### DISCUSSION TITLE
Logic Pro

### DISCUSSION THREAD

**Discussion by DJoumlala** [apohl79/audiogridder#898] [General]
Logic Pro 10.5.1 is not loading AudioGridder plugin 1.1.1
MacPro Mojave 10.14.6
On the MacPro (server) it is well installed
Any idea

**Comment by oldtos** [comment]
Be sure plugin install location is chosen for logic to look for plugin. 
The plugin will be listed as e47.

**Reply by DJoumlala** [reply]
Yes well locate with all other plug-ins
What is e47?

**Comment by DJoumlala** [comment]
finally I understood.
I have an issue.
On some tracks I can't make insert, I get your icon (snow star) and I get plugins in shadow grey

**Reply by DJoumlala** [reply]
i've found whyplug goes shadow grey, the channel was in freeze mode
